### DISCUSSION TITLE
How to get draft=true posts with lume

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [lumeland/lume#347] [Q&A]
I create a preview functionally with netlify cms. but I can't get all posts which not publish( draft =true)  yet. 

```
export const layout = "layouts/preview.njk";

export default function* (props) {

  const { search }= props


  for (const preview of search.pages("type=question", "draft=true") ) {

    const {slug} = preview.src
    
    yield {
      url: `/previews/${slug}/`,
      title: `previews ${slug}`,
      type:"previews",
      preview,
    };
  }
}
```
I also set `search.pages("type=question", "draft=true")` draft=true. it not working for me

**Comment by jrson83** [comment]
Did you try the `filter_pages` [plugin](https://lume.land/blog/posts/lume-1.14.0-release/#new-plugin-filter_pages)?

**Reply by officialrajdeepsingh** [reply]
I can't find ways how to use the `filter_pages` plugin

**Reply by oscarotero** [reply]
You can use the --dev flag to render all pages, including the draft ones.

```
deno task build --dev
```
